[2025-08-19T19:23:21.694+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_pipeline.data_validation manual__2025-08-19T19:22:42.304196+00:00 [queued]>
[2025-08-19T19:23:21.713+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_pipeline.data_validation manual__2025-08-19T19:22:42.304196+00:00 [queued]>
[2025-08-19T19:23:21.714+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-19T19:23:21.736+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): data_validation> on 2025-08-19 19:22:42.304196+00:00
[2025-08-19T19:23:21.743+0000] {standard_task_runner.py:60} INFO - Started process 319 to run task
[2025-08-19T19:23:21.752+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_pipeline', 'data_validation', 'manual__2025-08-19T19:22:42.304196+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn_pipeline_dag.py', '--cfg-path', '/tmp/tmp7jqv5r0q']
[2025-08-19T19:23:21.756+0000] {standard_task_runner.py:88} INFO - Job 38: Subtask data_validation
[2025-08-19T19:23:21.857+0000] {task_command.py:423} INFO - Running <TaskInstance: customer_churn_pipeline.data_validation manual__2025-08-19T19:22:42.304196+00:00 [running]> on host dc08f7cf7bac
[2025-08-19T19:23:21.999+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_pipeline' AIRFLOW_CTX_TASK_ID='data_validation' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T19:22:42.304196+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T19:22:42.304196+00:00'
[2025-08-19T19:23:22.004+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-19T19:23:22.009+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/Customer_Churn_Prediction_Pipeline && python src/data_validation.py']
[2025-08-19T19:23:22.044+0000] {subprocess.py:86} INFO - Output:
[2025-08-19T19:23:23.808+0000] {subprocess.py:93} INFO - === DATA VALIDATION REPORT ===
[2025-08-19T19:23:23.809+0000] {subprocess.py:93} INFO - === END OF REPORT ===
[2025-08-19T19:23:23.959+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-19T19:23:24.029+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=customer_churn_pipeline, task_id=data_validation, execution_date=20250819T192242, start_date=20250819T192321, end_date=20250819T192324
[2025-08-19T19:23:24.091+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-19T19:23:24.176+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
